.github/workflows/third-party-tests.yml

```diff
@@ -17,6 +17,9 @@ jobs:
           - 'php_parser'
           - 'symfony'
           - 'laravel'
+          - 'yii'
+          - 'mezzio'
+          - 'spiral'
```

Comment:
you should add silverstripe framework
They have some "weird" code in some areas

Reply:
would definitely accept a PR for it, see #184